packages/salesforcedx-vscode-apex/package.json

```diff
@@ -49,9 +52,89 @@
     "test": "node ./node_modules/vscode/bin/test",
     "test:unit": "node ./node_modules/vscode/bin/test"
   },
-  "activationEvents": ["workspaceContains:sfdx-project.json"],
+  "activationEvents": [
+    "workspaceContains:sfdx-project.json",
+    "onCommand:sfdx.force.test.view.showError",
```

Comment:
Why do we need to add this new activation events if the extension would already get activated if the project is in sfdx shape ?